packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/include/camera_avfoundation/CameraProperties.h

```diff
@@ -4,6 +4,7 @@
 
 @import AVFoundation;
 @import Foundation;
+@import UIKit;
```

Comment:
If I enable Swift Package Manager, I would get build errors like:

```
Unknown type name 'UIDeviceOrientation'
```

This change fixed these errors. It's not clear to me what's different between Cocoapods and SPM here.

 `UIDeviceOrientation` 
I'm not sure why this is necessary now